<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateTemplate</class>
 <widget class="QMainWindow" name="GenerateTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>97</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>97</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>531</width>
    <height>97</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generate Translation Template</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="generate">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模板生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_sce">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择剧情文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sce_route">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_text">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文本提取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回上级</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
